Makefile

```diff
@@ -240,7 +282,7 @@ docker-start-cluster-with-infura:
 	fi
 
 docker-start-cluster-with-datadog:
-	docker-compose -f docker-compose.yml -f docker-compose.dev.yml up plasma-contracts watcher childchain
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml up plasma-contracts watcher watcher_informational childchain
```

Comment:
Because if you want to get all *your* UTXOs, _watcher_ does not have an API to respond to that query, but *watcher_informational* does. And other APIs...

Reply:
But locally, you could run just *watcher_informational*. But swagger specification does not have both APIs.